di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/uatuser/IdentityMapper.java

```diff
@@ -78,6 +81,41 @@ public DisplayIdentity mapToDisplayable(Identity identity) {
 
     public SharedClaims mapToSharedClaim(Identity identity, boolean agedDOB) {
         FindDateOfBirth dateOfBirth = identity.findDateOfBirth();
+
+        List<CanonicalAddress> canonicalAddresses = new ArrayList<>();
+        for (UKAddress address : identity.UKAddress()) {
+            LocalDate validFrom = address.validFrom();
+            LocalDate validUntil = address.validUntil();
+
+            CanonicalAddress canonicalAddress =
+                    new CanonicalAddress(
+                            address.buildingNumber(),
+                            address.buildingName(),
+                            address.street(),
+                            address.townCity(),
+                            address.postCode(),
+                            // default / arbitrary value assigned for now as
+                            // the validFrom date is not available in test data
+                            validFrom,
+                            validUntil);
+            canonicalAddresses.add(canonicalAddress);
+        }
+
+        if (canonicalAddresses.get(0).validFrom() == null) {
```

Comment:
Remove this conditional statement code block 
Add the validFrom default date assignment logic above ( when mapping the identity.addresses() to List<CanonicalAddress>